ext_mongodb.php

```diff
@@ -210,7 +210,7 @@ function($value) {
 <<__NativeData("MongoDBDriverManager")>>
 class Manager {
 	<<__Native>>
-	public function __construct(string $dsn = "mongodb://127.0.0.1/", array $options = array(), array $driverOptions = array());
+	public function __construct(string $dsn = "", array $options = array(), array $driverOptions = array());
```

Comment:
Can you use `?string $dsn = null` here ([reference](https://docs.hhvm.com/hack/types/type-system#nullable))? PHPC actually defaults to null (zpp starts with `|s!`) and only falls back to `mongodb://127.0.0.1/` if the `char *` is null. An empty string will be sent along to libmongoc.